examples/shared_vpc/main.tf

```diff
@@ -14,28 +14,83 @@
  * limitations under the License.
  */
 
-locals {
-  cluster_type = "shared-vpc"
-}
+
 
 provider "google" {
   version = "~> 2.18.0"
-  region  = var.region
 }
 
+resource "random_string" "suffix" {
+  length  = 4
+  special = false
+  upper   = false
+}
+
+/************************************************
+	Networking in shared VPC to host cluster in
+ ***********************************************/
+
+locals {
+  gke_svpc_subnet     = "gke-svpc-main-${random_string.suffix.result}"
+  pods_gke_subnet     = "${local.gke_svpc_subnet}-pods"
+  services_gke_subnet = "${local.gke_svpc_subnet}-services"
+}
+
+// Inputs for Shared VPC aren't provided in reason it handled by VPC helper submodule
```

Comment:
`shared_vpc_host = true` should be set on this network module as it's not done in the helper submodule.